### PR TITLE
Move backend to new db

### PR DESCRIPTION
**Pull request by oscarahalvorsen**
### PR Type 🍏
What kind of change does this PR introduce?
```
[ ] Feature
[ ] Database
```
[Kanban Link](insert here)

### Context 🤷‍♀️
We needed new backend and calls. 

### What's new? 👶
I created a new backend with 3 tables: event, player and score. All these tables got their respective Api call named: getAllEvents, getAllPlayers and getAllScores. 
Event also got another Api call named getEventByEventName which is used now in frontend in the branch named: 
migrate-game-mode

NB! Must be merged together with: migrate-game-mode, which is in frontend
### Screenshots 🖼️
Add pictures if applicable.

### CODE REVIEW

**Review by Magnus-Farstad — APPROVED** [review]
Ser bra ut det her!

File: src/main/kotlin/org/entur/norgesturbackend/norgestur/controller/PlayerScoreController.kt
Comment (maribsta) [inline src/main/kotlin/org/entur/norgesturbackend/norgestur/controller/PlayerScoreController.kt:1]: Er dette kommentarer? De burde ikke bli med i mergen kanskje, hvis de er flyttet et annet sted.

File: src/main/kotlin/org/entur/norgesturbackend/norgestur/model/Event.kt
Comment (maribsta) [inline src/main/kotlin/org/entur/norgesturbackend/norgestur/model/Event.kt:1]: Ta bort TODO?

File: src/main/kotlin/org/entur/norgesturbackend/norgestur/controller/PlayerScoreController.kt
Comment (selmagudmundsen) [reply src/main/kotlin/org/entur/norgesturbackend/norgestur/controller/PlayerScoreController.kt:1]: Enig, lurt å slette kommentert kode.

**Comment by selmagudmundsen** [comment]
Syns det ser bra ut!! Godt arbeid!

File: src/main/kotlin/org/entur/norgesturbackend/norgestur/controller/PlayerScoreController.kt
Comment (oscarahalvorsen) [reply src/main/kotlin/org/entur/norgesturbackend/norgestur/controller/PlayerScoreController.kt:1]: OK!

File: src/main/kotlin/org/entur/norgesturbackend/norgestur/model/Event.kt
Comment (oscarahalvorsen) [reply src/main/kotlin/org/entur/norgesturbackend/norgestur/model/Event.kt:1]: ok borte

**Pull request merged**